src/js/pouch-express.js

```diff
@@ -200,22 +205,36 @@ gpii.pouch.express.cleanup = function (that) {
             cleanupPromises.push(databaseInstance.destroyPouch());
         });
 
+        var optionsFileCleanupPromise = fluid.promise();
+
+        fs.unlink(that.options.expressPouchConfigPath, function (error) {
+            if (error) {
+                optionsFileCleanupPromise.reject(error);
+            }
+            else {
+                optionsFileCleanupPromise.resolve();
+            }
+        });
+
+        cleanupPromises.push(optionsFileCleanupPromise);
+
         var cleanupSequence = fluid.promise.sequence(cleanupPromises);
         cleanupSequence.then(function () {
-            gpii.pouch.express.initExpressPouchdb(that);
-
-            gpii.pouch.express.initDbs(that).then(function () {
+            if (that.baseDirBelongsToUs) {
```

Comment:
Yes, thank you all, I'm glad you enjoyed it.  I decided to err on the side of also making it a meaningful and grammatically sound variable name, hence singular on "base" and plural on "belongs".  Even if you don't get the joke, it actually describes how the variable is used.